frontend/src/Onboarding/Onboarding.tsx

```diff
@@ -123,7 +131,16 @@ class OnboardingComponent extends React.Component<OnboardingProps> {
 
     return (
       <Container>
-        <Header></Header>
+        <Header>
+          <LoginLink
+            to={{ pathname: "/login", state: { userData: {} } }}
```

Comment:
What does the state value do here? (I'm more just curious)

Reply:
I believe it was just copied and pasted from somewhere else, so I think we can remove `state` here